py-rse-index.Rmd

```diff
@@ -72,100 +126,46 @@ Sami Virtanen
 
 ### Prerequisites
 
-Learners must be able to:
+Readers should already be using Python regularly for data analysis,
+and should be comfortable:
 
--   Write reusable functions to analyze tabular data using [R][r] or [Python][python].
--   Keep track of their work using [Git][git] and [GitHub][github].
--   Create reproducible reports using [R Markdown][r-markdown] or [Jupytext][jupytext] (in [JupyterLab Notebooks][jupyter]).
--   Publish those reports with appropriate metadata such as a [DOI][doi] and [ORCID][orcid].
+-   reading tabular data from files;
```

Comment:
This is an odd prerequisite because during this whole RSE volume we don't read any tabular data from files. I think this bullet point might also be a hangover from when we were listing the things that we did in the novice Python volume.